graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/internal/DgsSchemaProvider.kt

```diff
@@ -547,7 +547,9 @@ class DgsSchemaProvider(
         }
 
         schemas += metaInfSchemas
-        return schemas
+
+        val schemasFiltered = schemas.filter { it.toString().endsWith(".graphql]") || it.toString().endsWith(".graphqls]") }
```

Comment:
What is the `]` here for? 

You may also want to compare the lowercase version of the file name, if someone perhaps types ".Graphqls"

Reply:
The new commit uses the file name and is also case insensitive.